nuc_morph_analysis/lib/preprocessing/global_dataset_filtering.py

```diff
@@ -256,8 +261,29 @@ def process_full_tracks(df_all, thresh, pix_size, interval):
     df_full = add_growth_features.add_early_growth_rate(df_full, interval)
     df_full = add_growth_features.add_late_growth_rate_by_endpoints(df_full)
     df_full = add_growth_features.fit_tracks_to_time_powerlaw(df_full, "volume", interval)
-
+    
+    # For LRM
+    df_full = add_features.add_lineage_features(df_full, feature_list=['volume_at_B', 'duration_BC', 'volume_at_C', 'delta_volume_BC'])
+    
+    df_full = add_features.add_feature_at(df_full, "frame_transition", 'height', 'height_percentile', pix_size) 
+    for feature in ['xy_aspect', 'SA_vol_ratio', 'neighbor_avg_lrm_volume_90um', 'neighbor_avg_lrm_height_90um',
+                    'neighbor_avg_lrm_xy_aspect_90um','neighbor_avg_lrm_mesh_sa_90um']:
+        df_full = add_features.add_feature_at(df_full, "frame_transition", feature, feature)
+    
+    df_full = add_features.get_early_transient_gr_of_neighborhood(df_full, scale=get_plot_labels_for_metric('neighbor_avg_dxdt_48_volume_90um')[0])
     df_full = add_features.sum_mitotic_events_along_full_track(df_full)
+    df_full = add_features.normalize_sum_events(df_full, ['sum_has_mitotic_neighbor', 'sum_has_dying_neighbor'])
+    
+    ft_list = ['neighbor_avg_dxdt_48_volume_90um',
```

Comment:
it would be great to put lines 277-286 into a single function that is defined in `add_features`

Reply:
Done!